src/components/dashboard/GapByRace.vue

```diff
@@ -24,20 +24,21 @@
         />
       </div>
       <div class="centered">
-        <p class="has-text-centered">
-          In {{ activeCluster.name }}, the largest gap is among
-          {{ minVaxRace?.name }} residents. Only
-          <strong
-            >{{ formatPct(minVaxRace?.pct) }} of
-            {{ minVaxRace?.name }} residents</strong
-          >
-          are vaccinated compared to {{ formatPct(expected) }} statewide.
-          Approximately
-          <strong
-            >{{ minVaxRace?.gap }} more {{ minVaxRace?.name }} residents</strong
-          >
-          need to receive a dose to close this gap.
-        </p>
+        <!-- eslint-disable vue/no-v-html -->
```

Comment:
I remember thinking this for the last abstract text PR, but I wonder if it would be worth removing the `vue/no-v-html` rule from eslint on a global level? That way we wouldn't have to keep disabling it in the markup.